src/objective-c/GRPCClient/private/GRPCChannel.m

```diff
@@ -99,6 +100,22 @@ @implementation GRPCChannel {
   grpc_channel_args *_channelArgs;
 }
 
+- (instancetype)initWithHostUsingCronet:(NSString *)host
+                 engine:(void *)engine
+                 channelArgs:(NSDictionary *)channelArgs {
```

Comment:
A better name for this method would be `initWithHost:usingCronetEngine:channelArgs:`